scripts/gluster-block/setup-gluster-block-glusto.yml

```diff
@@ -0,0 +1,392 @@
+
+# Setup packages for gluster-block on client
+- hosts: gluster_nodes[5]
+  tasks:
+  - name: Install dependency packages
+    yum: name={{ item }} state=installed
+    with_items:
+    - iscsi-initiator-utils 
+    - device-mapper-multipath
+
+  - name: Start iscsi Daemon
+    service: name=iscsid state=started
+
+  - name: set multipath
+    command: "modprobe dm_multipath"
+
+  - name: Set multipath
+    command:  "mpathconf --enable"
+
+  - name: append the multipath.conf file
+    command: 'echo " # LIO iSCSI
+devices {
+        device {
+                vendor "LIO-ORG"
+                user_friendly_names "yes" # names like mpatha
+                path_grouping_policy "failover" # one path per group
+                path_selector "round-robin 0"
+                failback immediate
+                path_checker "tur"
+                prio "const"
+                no_path_retry 120
+                rr_weight "uniform"
+        }
+}" >> /etc/multipath.conf'
+
+  - name: Restart multipath daemon
+    service: name=multipathd state=restarted
+
+#peer probe all nodes
+- hosts: gluster_nodes[1]
+  tasks:
+  - name: Peer probe all nodes
+    command: "gluster peer probe {{hostvars[item]['ansible_eth0']['ipv4']['address']}}"
+    with_items: "{{groups['gluster_nodes'][2:4]}}"
+
+  - name: Install dependency packages
+    yum: name={{ item }} state=installed
+    with_items:
+    - ctdb
+    - samba
+    - samba-vfs-glusterfs
+    - samba-client
+
+  - name: Start smb
+    service: name=smb state=started
+
+  - name: Setup SELinux for Samba
+    seboolean: name=samba_load_libgfapi state=yes persistent=yes
+
+  - name: Setup SELinux in permissive mode
+    selinux: policy=targeted state=enforcing
+
+  - name: Setup Samba credentials
+    shell: (echo "foobar"; echo "foobar") | smbpasswd -a root
+  - name: Setup Samba configuration
+    blockinfile:
+      dest: /etc/samba/smb.conf
+      insertafter: '\[global\]'
+      content: |
+          kernel share modes = no
+          kernel oplocks = no
+          map archive = no
+          map hidden = no
+          map read only = no
+          map system = no
+          store dos attributes = yes
+      tags: smb
+
+
+# Set up glusto packages on the host node
+- hosts: gluster_nodes[0]
+  tasks:
+  - name: Copy ssh key
+    copy: src="{{ lookup('env', 'GLUSTO_WORKSPACE') }}/glusto" dest=/root/.ssh/id_rsa mode=600
+
+  - name: Install git and pip
+    yum: name="{{item}}" state=present
+    with_items:
+    - git
+    - python-pip
+
+  - name: Set glusto version
+    set_fact:
+      GERRIT_REFSPEC: "{{ lookup('env', 'GERRIT_REFSPEC') }}"
+
+  - name: Install Glusto
+    pip: name=' git+git://github.com/loadtheaccumulator/glusto.git' editable=false
+
+  - name: Clone the glusto-tests repo - {{ GERRIT_REFSPEC}}
+    git:
+        repo: git://review.gluster.org/glusto-tests.git
+        dest: /root/glusto-tests
+        refspec: "+refs/heads/master:refs/remotes/origin/master"
```

Comment:
If you want to use `GERRIT_REFSPEC` at all, you should put that variable here. Otherwise skip the entire process of even getting that variable as a config.